recipes/cvxopt/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "1.1.8" %}
+
+{% set variant = "openblas" %}
+
+
+package:
+  name: cvxopt
+  version: {{ version }}
+
+source:
+  fn: cvxopt-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/c/cvxopt/cvxopt-{{ version }}.tar.gz
+  md5: d4ff0a0e51aba93f838a70302a088eea
+
+build:
+  skip: true  # [win]
+  number: 200
```

Comment:
Yeah, this is some funny business that @pelson and I discovered while we tried to figure out how to do BLAS. We wrote it up in a [hackpad]( https://conda-forge.hackpad.com/BLAS-Numpy-Friends-86De62hoNdT ). The point about using `200` was initially controversial, but we settled it in a later [discussion]( https://github.com/conda-forge/numpy-feedstock/pull/6#issuecomment-220762038 ).